swagger-spec/institutions/nodes_relationships.yaml

```diff
@@ -1 +1,76 @@
 # /institutions/{institution_id}/relationships/nodes/
+get:
+  summary: List all institution-node relationships
+  description: >-
+    Returns all nodes affiliated with an institution. Users that have an institutional
+    affiliation can add the affiliation to nodes that they have write access over. This
+    displays an institutional logo on the node page.
+
+  tags:
+    - Institutions
+  operationId: institutions_relationships_nodes
+  responses:
+    '200':
+      description: OK
+      schema:
+        type: array
+        items:
+        # It is not clear what this should be at the moment
```

Comment:
I think it might be better to ignore an actual definition here and better document in the description what the response body looks like.